deno/src/platform.ts

```diff
@@ -46,7 +48,11 @@ export const inputFileData = Symbol('InputFile data')
  * Reference](https://core.telegram.org/bots/api#inputfile).
  */
 export class InputFile {
-    public readonly [inputFileData]: ConstructorParameters<typeof InputFile>[0]
+    private [internalInputFileData]: ConstructorParameters<typeof InputFile>[0]
+    public get [inputFileData]() {
+        const file = this[internalInputFileData]
+        return file instanceof File ? file.stream() : file
```

Comment:
```suggestion
        return file instanceof Blob ? file.stream() : file
```

I feel this getter can be simplified, but I'm not yet sure how...

Reply:
I don't see any possible simplification either, especially because narrowing the type does not work with bracket access. Do let me know if you find something :)